docs/ja/about-appium/intro.md

```diff
@@ -0,0 +1,138 @@
+
+1. アプリを自動化するために、
+アプリを再コンパイルしたり、修正したりする必要はありません
+2. テストを書いて実行するために、
+特定の言語やフレームワークに縛られるべきではありません
+3. モバイルオートメーションフレームワークは、
+自動化 API に関しては、車輪の再発明をしてはいけません
+4. モバイルオートメーションフレームワークは、名前だけでなく、
+精神的にも実践的にもオープンソースでなければなりません!
+
+### Appium のデザイン
+
+では、Appium プロジェクトの構造はどのようにしてこの理念を実現しているのでしょうか？
+ベンダーが提供する自動化フレームワークをアンダーグラウンドで使用することで、要件 1 を満たしています。
+この方法では、Appium 固有のコードやサードパーティのコードやフレームワークをアプリにコンパイルする必要はありません。
+つまり、**出荷するアプリと同じアプリをテストしていることになります**。
+使用しているベンダー提供のフレームワークは以下の通りです。
+
+* iOS 9.3 以上: Apple の [XCUITest](https://developer.apple.com/reference/xctest)
+* iOS 9.3 以下: Apple の [UIAutomation](https://web.archive.org/web/20160904214108/https://developer.apple.com/library/ios/documentation/DeveloperTools/Reference/UIAutomationRef/)
+* Android 4.3 以上: Google の [UiAutomator/UiAutomator2](https://developer.android.com/training/testing/ui-automator)
+* Windows: Microsoft の [WinAppDriver](http://github.com/microsoft/winappdriver)
+
+ベンダーが提供するフレームワークを
+[WebDriver](http://docs.seleniumhq.org/projects/webdriver/) API という1つの API でラップすることで要件2を満たしています。
+WebDriver (別名 "Selenium WebDriver") は、([JSON Wire Protocol](https://w3c.github.io/webdriver/webdriver-spec.html) として知られる) 
+クライアント - サーバプロトコルを指定します。
+このクライアント - サーバアーキテクチャを考えると、任意の言語で書かれたクライアントを使用して、
+適切な HTTP リクエストをサーバに送信することができます。
+既に[あらゆる一般的なプログラミング言語で書かれたクライアント](http://appium.io/downloads)
+が存在します。
+これはまた、好きなテストランナーやテストフレームワークを
+自由に使用できることを意味します。
+言い換えれば、Appium と WebDriver クライアントは技術的には「テストフレームワーク」ではなく、
+「自動化ライブラリ」です。
+テスト環境を
+好きなように管理することができます。
+
+同様に要件3を満たしています。
+WebDriver は、Web ブラウザを自動化するためのデファクトスタンダードとなっており、
+[W3C のワーキングドラフト](https://dvcs.w3.org/hg/webdriver/raw-file/tip/webdriver-spec.html)です。
+なぜモバイル用に全く違うことをするのでしょうか?
+その代わりに、モバイルの自動化に役立つ API メソッドを追加して
+[プロトコルを拡張](https://github.com/SeleniumHQ/mobile-spec/blob/master/spec-draft.md)しました。
+
+要件4が与えられたものであることは明らかなはずです。
+あなたがこれを読んでいるのは、[Appium がオープンソース](https://github.com/appium/appium)だからです。
+
+### Appium の概念
+
+#### **クライアント/サーバー アーキテクチャ**
+
+Appium は、REST API を公開する Web サーバーです。
+クライアントからの接続を受信し、コマンドを待ち受け、
+モバイル デバイス上でそれらのコマンドを実行し、コマンドの実行結果を表す HTTP レスポンスで応答します。
+クライアント/サーバー アーキテクチャがあるという事実は、多くの可能性を開きます。
+http クライアント API を持つどの言語でもテストコードを書くことができますが、
+[Appium クライアントライブラリ](http://appium.io/downloads)の 1 つを使用する方が簡単です。
+テストが実行されているマシンとは別のマシンにサーバーを置くことができます。
+テストコードを書き、コマンドを受信して解釈するために 
+[Sauce Labs](https://saucelabs.com/products/mobile-app-testing) や [LambdaTest](https://www.lambdatest.com/feature) のようなクラウドサービスに頼ることもできます。
+
+#### **セッション**
+オートメーションは常にセッションのコンテキストで実行されます。
+クライアントはそれぞれのライブラリに固有の方法でサーバとのセッションを開始しますが、
+最終的にはサーバに `POST /session` リクエストを送信し、'desired capabilities' オブジェクトと呼ばれる JSON オブジェクトを送信します。
+この時点で、サーバーはオートメーションセッションを起動し、
+さらにコマンドを送信するために使用されるセッション ID で応答します。
+
+#### **Desired capabilities**
+Desired capabilities とは、どのような種類の自動化セッションの起動に興味があるかをサーバーに伝えるために、
+Appium サーバーに送られるキーと値のセット (すなわち、マップやハッシュ) です。
+また、自動化中にサーバーの動作を変更できる様々な機能があります。
+例えば、Android や Windows のセッションではなく、
+iOS のセッションが必要であることを Appium に伝えるために、
+`platformName` ケイパビリティを `iOS` に設定することができます。
```

Comment:
In Appium Desktop, we don't use this 'capability' word as katakana since _desired capabilities_ etc also remains as alphabets.

```suggestion
`platformName` capability を `iOS` に設定することができます。
```